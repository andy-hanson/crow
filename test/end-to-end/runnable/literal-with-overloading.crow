import
	crow/io/print: out, print

main void() summon
	out print 1.f
	out print "1".f
	()

f string(a int)
	"{a} int"
f string(a string)
	"{a} string"
