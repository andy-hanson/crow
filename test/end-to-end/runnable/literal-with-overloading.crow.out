1 int
1 string
